lib/ain-ocean/src/model/raw_tx.rs

```diff
@@ -0,0 +1,63 @@
+use bitcoin::{address::NetworkUnchecked, Address, Amount, BlockHash, Txid};
+use defichain_rpc::json::GetTransactionResultDetailCategory;
+use rust_decimal::Decimal;
+use serde::{Deserialize, Serialize};
+#[derive(Serialize, Deserialize, Default, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct RawTxDto {
+    pub hex: String,
+    pub max_fee_rate: Option<Decimal>,
+}
+
+pub fn default_max_fee_rate() -> Option<Amount> {
+    let default_max_fee_rate = Amount::from_btc(0.1).unwrap_or_default();
+    Some(default_max_fee_rate)
+}
```

Comment:
i think the `Decimal` is good enough to fulfill the bitcoin::Amount
just make it simple

Reply:
I reference from https://github.com/DeFiCh/rust-defichain-rpc/blob/main/integration_test/src/main.rs 
`static ref FEE: Amount = Amount::from_btc(0.001).unwrap();`

Reply:
can just do `const MAX_FEE_RATE = dec!(0.1)` as simple as possible